TDD-IntegrationTests/HealthCheckIntegrationTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using System.Net;
+using TDDProject;
```

Comment:
Add all the namespaces inside globalusings file

Reply:
Move the common namespaces to the global using file